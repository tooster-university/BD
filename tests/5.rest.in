{"open": {"database": "student", "login": "app", "password": "qwerty"}}
{"protest": {"timestamp": 63958382, "member": 7, "password": "R3UC0", "action": 183, "project": 356, "authority": 407}}
{"actions": {"timestamp": 83302507, "member": 9, "password": "6DVDR"}}
{"trolls": {"timestamp": 88194832}}
{"protest": {"timestamp": 105294706, "member": 10, "password": "P6H1AD9K", "action": 121, "project": 310, "authority": 434}}
{"votes": {"timestamp": 135217369, "member": 8, "password": "QV18921J"}}
{"upvote": {"timestamp": 137517151, "member": 3, "password": "UEUCC863NS", "action": 104}}
{"downvote": {"timestamp": 170142689, "member": 10, "password": "P6H1AD9K", "action": 129}}
{"support": {"timestamp": 209706786, "member": 9, "password": "6DVDR", "action": 182, "project": 332, "authority": 397}}
{"actions": {"timestamp": 251173748, "member": 8, "password": "QV18921J"}}
{"projects": {"timestamp": 298664235, "member": 10, "password": "P6H1AD9K"}}
{"trolls": {"timestamp": 312809305}}
{"downvote": {"timestamp": 363278193, "member": 10, "password": "F6SCROBC10", "action": 119}}
{"trolls": {"timestamp": 390506312}}
{"protest": {"timestamp": 446627967, "member": 9, "password": "6DVDR", "action": 184, "project": 319, "authority": 407}}
{"trolls": {"timestamp": 472342498}}
