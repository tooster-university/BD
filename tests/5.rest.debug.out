{"status": "OK"}
{"status": "OK"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "OK", "data": []}
{"status": "OK"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "ERROR", "debug": "User can't cast vote on action that does not exist!"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "OK"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "OK", "data": []}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "OK", "data": []}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
{"status": "OK", "data": []}
